src/components/common/PageHeader/index.tsx

```diff
@@ -1,10 +1,13 @@
 import useAddressBook from '@/hooks/useAddressBook'
 import useSafeAddress from '@/hooks/useSafeAddress'
-import { Box, Typography } from '@mui/material'
+import { Box, Typography, useMediaQuery } from '@mui/material'
 import type { ReactElement } from 'react'
 
 import css from './styles.module.css'
 
+const SIDEBAR_HEADER_HEIGHT = 199
+const RETINA_DISPLAY_MEDIA_QUERY = '(-webkit-min-device-pixel-ratio: 2), (min-resolution: 192dpi)'
```

Comment:
Can we do it in CSS instead? A padding instead of adding to the height. (You'll need to adjust the box-sizing too).

Reply:
I've changed it. Please confirm if it works.

Reply:
Now the page header is 1px lower :(